### ISSUE TITLE
Searching for Chinese Supplier for Heater Quartz Tubes

### ISSUE THREAD

**Issue by Hayleene** [Hayleene/Perosnal-Finance#4]
Hello everyone, 

I'm looking for a Chinese supplier of Heater Quartz Tubes that caters specifically to businesses rather than individual consumers. If anyone has any leads or recommendations, please share. I need a reliable company with high-quality products and good customer service. It's crucial for our project, and I would appreciate any suggestions or personal experiences with suppliers in this industry. 

Thank you in advance!

**Comment by Lucassane** [comment]
Google is your best friend mate. You can search on browser but if you're looking for hands on experience of someone with any manufacturer or supplier, then you've to wait.

**Comment by Hayleene** [comment]
Yes, I wanna get suggestions from those who've experienced with any supplier.

**Comment by MikeColady** [comment]
> Yes, I wanna get suggestions from those who've experienced with any supplier.

You might want to check out Global Quartz Tube, a reputable Chinese manufacturer that specializes in supplying Heater Quartz Tubes to businesses. They are known for their high-quality products and excellent customer service. You can reach them through their website: [https://globalquartztube.com/heater-quartz-tube/](https://globalquartztube.com/heater-quartz-tube/). Hope this helps with your project!
